plugins/Jobs.cpp

```diff
@@ -367,8 +372,9 @@ bool BedrockPlugin_Jobs::processCommand(BedrockNode* node, SQLite& db, BedrockNo
         // Update the data
         if (!db.write("UPDATE jobs "
                       "SET data=" +
-                      SQ(request["data"]) + " "
-                                            "WHERE jobID=" +
+                      SQ(request["data"]) + " " +
+                      (request.isSet("repeat") ? ", repeat=" + SQ(SToUpper(request["repeat"])) : "") +
```

Comment:
No. How could you get `, repeat=` without matching the truth test of the ternary? (also, this is identical to other code in this file that uses a ternary to update optional information)